src/Mysql.php

```diff
@@ -30,11 +30,13 @@ class Mysql
      * @param $database
      * @param int $fetchMode
      * @param string $charset
+     * @param array $pdo_options
      * @param array $options
      *
      * @throws MysqlException
      */
-    public function __construct($host, $user, $password, $database, $fetchMode = \PDO::FETCH_ASSOC, $charset = 'utf8', array $options = [])
+    public function __construct($host, $user, $password, $database, $fetchMode = \PDO::FETCH_ASSOC,
+                                $charset = 'utf8', array $pdo_options = array(), array $options = [])
```

Comment:
Have a look at the latest version (1.4.1). I added the possibility to inject PDO options. Have a look at point https://github.com/fightbulc/simplon_mysql#3-setup-connection to see on how to to do this.